momentum/4/modules/ha-proxy-client.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "ha_proxy_client - HAProxy protocol client module"
+description: "The ha_proxy_client module is used to configure Momentum to use HAProxy's PROXY protocol version 2 for outbound connections (see https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt).  This can be leveraged in cases where your sending IPs are deployed on a different machine than the Momentum.
```

Comment:
This was probably me but I don't think we should say `than the Momentum`, think it should just be `than Momentum`